background.js

```diff
@@ -377,6 +377,7 @@ async function on_message_native(response) {
         } else {
             ssoLog("lost connection to broker");
             broker_online = false;
+            logout();
```

Comment:
This breaks on-demand dbus activation and was removed in https://github.com/siemens/linux-entra-sso/commit/04c06aab8a95a91abef73b6e1424b17182d14cd8#diff-bb06783fccd762e4d905ab922491e954a9fbaf708a1acbf53dff92872b0bcd3bL335.

For details, see #33.

Reply:
But it's still broken. We need to signal that the broker is offline, also visually.

Reply:
Not necessarily. It is fine if the broker is offline as it will get restarted on the next DBus call. If we logout, the next hit of the Entra login page will not even try to start it.

Reply:
But visualization is broken - I will fix that instead.

Reply:
We might give a slight indication that the broker is offline. But in the end, the user should not need to care and should also not be disturbed by that. For (caller activated) DBus services the caller should not assume anything about the current state of the service. Instead it should just make the call and check the return code - which indicates the error reason.